app/src/main/java/com/ghomebyrw/gworker/models/Job.java

```diff
@@ -20,23 +20,23 @@
     private HashMap<String, String> questionToAnswer;
     private String fieldworker;
 
-    public Job(String serviceName, ScheduledDateAndTime scheduledDateAndTime){
-//               JobStatus status,
-//               UUID customerId, Price acceptedPrice, String location,
-//               String timeZone, String customerPhoneNumber, String note, int estimatedMinutes,
-//               HashMap<String, String> questionToAnswer, String fieldworker) {
+    public Job(String serviceName, ScheduledDateAndTime scheduledDateAndTime,
+               JobStatus status,
+               UUID customerId, Price acceptedPrice, Location location,
+               String timeZone, String customerPhoneNumber, String note, int estimatedMinutes,
+               HashMap<String, String> questionToAnswer, String fieldworker) {
```

Comment:
I'm not a big fan of constructors (or methods) that take tons of params. If this object is just a dumb holder of data, I'd rather set the data via public fields or setters. Not a blocker though.